tools/templates/assets/css/samples.css

```diff
@@ -0,0 +1,198 @@
+/* General ********************************************************************/
+
+html, body {
+  height: 100%;
+}
+
+a:link, a:visited, a:active {
+  color: #4183C4;
+}
+
+#samples, #samples .ui.menu, #samples .ui.table {
+  font-size: 15px;
```

Comment:
Ideally, yes. There's some inconsistency in the classes I'm trying to override (depending on whether the styles are defined in the widget, courage, or semantic-ui). I'll take another pass at it.